.github/workflows/updatecli.yml

```diff
@@ -17,6 +17,18 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Get token
```

Comment:
```
permissions:
  contents: read
```

Do we need to update this instead and rely on `GITHUB_TOKEN`?

Reply:
Unfortunately, GitHub actions don't trigger any action if the `Actor` is `Github Actions bot`. 

See https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret

> Commits pushed by a GitHub Actions workflow that uses the GITHUB_TOKEN do not trigger a GitHub Pages build.

That's the reason we use a Service Machine account, and we are moving to a GitHub app to help us moving away from finer-tokens and rotating them.